.github/workflows/main.yml

```
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: ['1.0', '1.1', '1.2', '1.3', '1.4', '1.5', '1.6', '1.7', '1.8', '1.9']
+        version: ['1.6', '1.7', '1.8', '1.9']
```

It's possible to safe some compute time by setting only the following

```suggestion
        version:
          - '1.6'
          - '1' # Latest stable release.
```
This is fairly common in the Julia ecosystem, see for example https://github.com/JuliaData/DataFrames.jl/blob/main/.github/workflows/ci.yml.

It assumes that if 1.6 works and the latest stable works (1.9.3 currently), then probably everything in between does as well.